common.blocks/issue/issue.css

```diff
@@ -50,3 +51,43 @@
 {
     border-bottom: 1px solid #e1e1e1;
 }
+
+.issue .button__control {
```

Comment:
Fixed with mix issue__middlerow-button. Variant with common middlerow wrapper was refused, because comment button can be replaced with comments block.